test/whl-scale-tests/crash-dumps/deploy.ps1

```diff
@@ -0,0 +1,50 @@
+<#
+.SYNOPSIS
+Deploy a crash dumps DaemonSet
+.DESCRIPTION
+Build and deploy a DaemonSet to generate crash dumps
+.PARAMETER ImageTag
+Full tag to use for the container image in the format <uri>/<tag>:<version>
+.PARAMETER name
+Name to give the DaemonSet
+.PARAMETER namespace
+Kubernetes namespace where the DaemonSet should be deployed
+.PARAMETER nodeSelector
+String to match for selecting nodes where the DaemonSet should be deployed
+.EXAMPLE
+Deploy-LogGenerator exampleacr.azurecr.io/generatelogs:latest LogGenerator log-generation whl-logs
+#>
+param(
+  [Parameter(Mandatory = $true)][string] $ImageTag,
+  [Parameter(Mandatory = $true)][string] $WindowsVersion,
+  [Parameter(Mandatory = $true)][string] $namespace,
+  [Parameter(Mandatory = $true)][string] $nodeSelector
+)
+
+. (Join-Path $PSScriptRoot "../common.ps1")
+
+$NUGET_SOURCE = "https://msazure.pkgs.visualstudio.com/One/_packaging/OneBranch-Consumption/nuget/v3/index.json"
+$PACKAGE_NAME = "AzwHugeDump-retail-amd64"
+$PACKAGE_VERSION = "1.0.97"
+$DownloadName = "$PACKAGE_NAME.$PACKAGE_VERSION" # Expected folder/file name of the downloaded nuget package
+$TmpDir = Get-TempDir
+$DownloadPath = Join-Path $TmpDir "$DownloadName\$DownloadName.nupkg"
+$TmpZipFile="$PSScriptRoot\crashdumpgenerator.zip" # Can't go in temp directory since dockerfile won't have access
```

Comment:
If we pass in a different context we should be able to see the TempDir. Also since you are creating the temp dir you can put it where it makes the most sense to improve the usage of the automation.